### ISSUE TITLE
[T12][PowerChart][Functional][Accessibility] Focus is missing on first tab in Histories Tab

### ISSUE THREAD

**Issue by shiva87** [Mohanme13/Feb-19-Bug-bash#94]
Steps to reproduce:
1. Open Powerchart to test, testing patient.
2. Navigate to Histories tab
